_grants/july-2016-uniform-guidance-promising-practices-in-implementation.md

```diff
@@ -1,11 +1,16 @@
 ---
```

Comment:
This file needs to be changed under policies_guidance folder. The grants, ERM, FDT, FMT etc. folders will ultimately be deleted as a part of refactoring.